tasks/fauxton.js

```diff
@@ -59,7 +59,7 @@ module.exports = function (grunt) {
     grunt.log.writeln(localDeps.length + ' local dependencies');
     var local = fetch(localDeps, function (dep, destination) {
       // TODO: Windows
```

Comment:
I can't wait.

On Tue, Apr 5, 2016 at 2:40 PM, Robert Kowalski <notifications@github.com>
wrote:

> In tasks/fauxton.js
> <https://github.com/apache/couchdb-fauxton/pull/677#discussion_r58530207>:
>
> > @@ -59,7 +59,7 @@ module.exports = function (grunt) {
> >      grunt.log.writeln(localDeps.length + ' local dependencies');
> >      var local = fetch(localDeps, function (dep, destination) {
> >        // TODO: Windows
>
> everythign will get better:
> https://blogs.windows.com/buildingapps/2016/03/30/run-bash-on-ubuntu-on-windows/
>
> :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/couchdb-fauxton/pull/677/files/ef286dad1a0f2085cd83e60d539b448a1b6d606f#r58530207>
>